app/templates/contact/index.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/page_macros.html' as page %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui text container">
+        <h1>Contact Us</h1>
+        {% set flashes = {
+            'error': get_flashed_messages(category_filter=['form-error']),
+            'warning': get_flashed_messages(category_filter=['form-check-email']),
+            'info': get_flashed_messages(category_filter=['form-info']),
+            'success': get_flashed_messages(category_filter=['form-success'])
+            }
+        %}
+        {% if current_user.is_admin() %}
+            <h3>Edit Contact Description</h3>
+        {% endif %}
+        {{ page.render_inline_editor(editable_html_obj, current_user) }}
+        {% if not current_user.is_admin() %}
+
```

Comment:
Tried doing this with ui horizontal divider but didn't work without adding text. How can I do this?